src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/EventHandlers/OverlappingTrainingDateRequest/OverlappingTrainingDateEditEndDateEventHandler.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.Extensions.Logging;
+using NServiceBus;
+using SFA.DAS.CommitmentsV2.Configuration;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Data.Extensions;
+using SFA.DAS.CommitmentsV2.Messages.Commands;
+using SFA.DAS.CommitmentsV2.Models;
+using SFA.DAS.Encoding;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events.OverlappingTrainingDateRequest
+{
+    public class OverlappingTrainingDateEditEndDateEventHandler : IHandleMessages<OverlappingTrainingDateEvent>
```

Comment:
This same event is being handled by `OverlappingTrainingDateEventHandler.cs`. 

If retaining the separate event handlers, then we should: Ensure that both handlers are named appropriately i.e `OverlappingTrainingDateForCompletedApprenticeshipEventHandler` and as appropriate for the inverse

Reply:
Done